### ISSUE TITLE
Open two python sessions inside R?

### ISSUE THREAD

**Issue by saptarshiguha** [rstudio/reticulate#341]
Hello,
Is it possible to embed two python sessions inside R? Calling `library(reticulate)` automatically embeds one. Can I embed two?

Much thanks
Saptarshi

**Comment by jjallaire** [comment]
No, this isn't possible (nor will it ever be as there is one embedded
Python interpreter per-process).
